app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,16 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+  end
+
+  def update
```

Comment:
Like we talked about in office hours, this action would be best reserved for updating a movie's attributes. A more RESTful option would be to route to a MovieActorsController #create action